pallets/author-noting/src/lib.rs

```diff
@@ -195,14 +197,15 @@ pub mod pallet {
         /// Latest author changed
         LatestAuthorChanged {
             para_id: ParaId,
+            block_number: BlockNumber,
             new_author: T::AccountId,
         },
     }
 
     #[pallet::storage]
     #[pallet::getter(fn latest_author)]
     pub(super) type LatestAuthor<T: Config> =
-        StorageMap<_, Blake2_128Concat, ParaId, T::AccountId, OptionQuery>;
+        StorageMap<_, Blake2_128Concat, ParaId, (BlockNumber, T::AccountId), OptionQuery>;
```

Comment:
I would make a new struct instead of a tuple, I think it makes things more clear. Something like containerChainBlockInfo